src/index.js

```diff
@@ -71,6 +72,10 @@ export default function (debug = printDebug) {
       })
     }
 
+    function buildAssertionsOutOfWrapperObject(wrapperObject) {
+      Object.keys(wrapperObject).forEach(key => overwriteMethod(wrapperObject[key], key))
```

Comment:
rather than requiring ES5, it'd be good to use the `object-keys` or `object.entries` package, and the `foreach` package.